fuel-vm/src/interpreter/contract.rs

```diff
@@ -409,26 +409,29 @@ where
 }
 
 /// Increase the asset balance for a contract.
-/// Returns new balance, and a boolean indicating if a new entry was created.
+/// A boolean indicating if a new entry was created.
 pub fn balance_increase<S>(
     storage: &mut S,
     contract: &ContractId,
     asset_id: &AssetId,
     amount: Word,
-) -> IoResult<(Word, bool), S::Error>
+) -> IoResult<bool, S::Error>
```

Comment:
Since we change the public interface, I suppose this is a breaking change. Do we have any idea of potential downstream impact of this?